.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "PyLaunch Blueprint",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "context": ".."
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-python.python",
+        "ms-python.vscode-pylance",
+        "ms-python.black-formatter",
+        "charliermarsh.ruff",
+        "matangover.mypy",
+        "tamasfe.even-better-toml",
+        "redhat.vscode-yaml",
+        "eamodio.gitlens",
+        "streetsidesoftware.code-spell-checker"
+      ]
+    }
+  },
+  "postCreateCommand": "bash .devcontainer/setup.sh",
```

Comment:
Where does the post create command get run? Does it get run on the host machine or does it get run in the Docker container? Please clarify.